.github/data/maintainers.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Is it possible to store this in env variable or repo variable?
So that further maintenance does not require code change.

Reply:
Removed maintains info and put in environment variable